src/main/main.go

```diff
@@ -27,44 +25,11 @@ func main() {
 		config.GlobalConfig,
 		ops.NewOps(logger),
 		inventory_client.CreateBmInventoryClient(),
-		getKubeClient(logger),
+		k8s_client.NewK8SClient,
 	)
 	if err := installer.InstallNode(); err != nil {
 		installer.UpdateHostStatus(fmt.Sprintf("%s %s", Failed, err))
 		os.Exit(1)
 	}
 	installer.UpdateHostStatus(Done)
 }
-
-func getKubeClient(logger *logrus.Logger) k8s_client.K8SClient {
-	var kc k8s_client.K8SClient
-	// in case of bootstap download the kubeconfig and create the k8s client
-	if config.GlobalConfig.Role == installer.HostRoleBootstrap {
-		kubeconfigPath, err := getKubeconfig(logger)
-		if err != nil {
-			logrus.Fatal(err)
-		}
-		kc, err = k8s_client.NewK8SClient(kubeconfigPath, logger)
-		if err != nil {
-			logrus.Fatal(err)
-		}
-	}
-	return kc
-}
-
-func getKubeconfig(logger *logrus.Logger) (string, error) {
-	o := ops.NewOps(logger)
-	ic := inventory_client.CreateBmInventoryClient()
-	if err := o.Mkdir(installer.InstallDir); err != nil {
-		logger.Errorf("Failed to create install dir: %s", err)
-		return "", err
-	}
-	filename := "kubeconfig"
-	dest := filepath.Join(installer.InstallDir, filename)
-	err := ic.DownloadFile(filename, dest)
```

Comment:
Is there no need to download the kubeconfig anymore?

Reply:
No, we use the one we get from the ignition (kubeconfig-loopback)

Reply:
So who uses the kubeconfig that is in the cloud?

Reply:
The user